### PR TITLE
[docs]: Add release instructions

### PR DESCRIPTION
**Pull request by genevieveluyt**
Adds instructions on how to make a release.

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Since Goreleaser is set to draft mode, it will build Docker images but not publish them. You can see the images with `docker images`.
+
+Login to the Shopify Docker account using `docker login`. Find the credentials in password manager.
+
+Push up the following three images:
+
+- latest: `shopify/kubeaudit:latest`
+- patch: `shopify/kubeaudit:<VERSION>` for example `shopify/kubeaudit:v0.11.6`
+- minor: `shopify/kubeaudit:v<MAJOR>.<MINOR>` for example `shopify/kubeaudit:v0.11`
```
Comment (dani-santos-code) [inline docs/release.md:61]: shouldn't this be:

- latest: `shopify/kubeaudit:latest`
- patch: `shopify/kubeaudit:<VERSION>` for example `shopify/kubeaudit:v0.11`
- minor: `shopify/kubeaudit:v<MAJOR>.<MINOR>` for example `shopify/kubeaudit:v0.11.16` ?

**Review by dani-santos-code — COMMENTED** [review]
It looks really good! Very detailed and well explained with helpful links and examples! 🎉

**Review by dani-santos-code — APPROVED** [review]

File: docs/release.md (hunk 1 above)
Comment (genevieveluyt) [reply docs/release.md:61]: The patch version is the third number and the minor is the second number. Is expanding out `<VERSION>` less confusing? Like

```
- latest: `shopify/kubeaudit:latest`
- patch: `shopify/kubeaudit:<MAJOR>.<MINOR>.<PATCH>` for example `shopify/kubeaudit:v0.11.6`
- minor: `shopify/kubeaudit:v<MAJOR>.<MINOR>` for example `shopify/kubeaudit:v0.11`
```

Comment (dani-santos-code) [reply docs/release.md:61]: yeah! I guess that makes it clearer!

**Review by dani-santos-code — APPROVED** [review]

**Pull request merged**